mifospay/src/main/java/org/mifos/mobilewallet/mifospay/kyc/ui/KYCLevel1Screen.kt

```diff
@@ -0,0 +1,239 @@
+                address1 = it },
+            label = {
+                Text(stringResource(R.string.address_line_1)) },
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(vertical = 7.dp)
+        )
+        OutlinedTextField(
+            value = address2,
+            onValueChange = {
+                address2 = it },
+            label = {
+                Text(stringResource(R.string.address_line_2)) },
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(vertical = 7.dp)
+        )
+
+        Box (
+            modifier = Modifier
+                .padding(vertical = 7.dp)
+        ){
+            val keyboardController = LocalSoftwareKeyboardController.current
+            TogiCountryCodePicker(
+                modifier = Modifier,
+                shape = RoundedCornerShape(3.dp),
+                colors = TextFieldDefaults.outlinedTextFieldColors(
+                    focusedBorderColor = MaterialTheme.colorScheme.primary
+                ),
+                onValueChange = { (code, phone), isValid ->
+                    if (isValid){
+                        mobileNumber = code + phone
+                    }
+                },
+                label = {
+                    Text(stringResource(id = R.string.phone_number)) },
+                keyboardActions = KeyboardActions { keyboardController?.hide() }
+            )
+        }
+
+        CalendarDialog(
+            state = dateState,
+            config = CalendarConfig(
+                monthSelection = true,
+                yearSelection = true,
+                style = CalendarStyle.MONTH
+            ),
+            selection = CalendarSelection.Date{ date ->
+                dateOfBirth = dateFormatter.format(date)
+            }
+        )
+
+        Box(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(vertical = 9.dp)
+                .clickable { dateState.show() }
+                .border(
+                    width = 1.dp,
+                    color = Color.Black
+                )
+                .padding(16.dp)
+                .clip(shape = RoundedCornerShape(8.dp))
+        ) {
+            Text(
+                text = dateOfBirth.ifEmpty { stringResource(R.string.select_dob) },
+                style = MaterialTheme.typography.bodyLarge
+            )
+        }
+        Button(
+            onClick = {
+                submitData(
+                    firstName,
+                    lastName,
+                    address1,
+                    address2,
+                    mobileNumber,
+                    dateOfBirth
+                )},
+            modifier = Modifier
+                .align(Alignment.CenterHorizontally)
+                .padding(vertical = 7.dp),
+        ) {
+            Text(text = stringResource(R.string.submit))
+        }
+    }
+}
+
+@Preview(showBackground = true)
+@Composable
```

Comment:
shouldn't we add preview for kycscreen with all other ui states

Reply:
They are Toasts and screen will navigate back after showing them so I didn't add them, But I think I will add them now. And plus is there any way to preview Togicountrycodepicker , I have researched everywhere but I didn't find any solution yet. Is there any other maintained library available for this.

Reply:
I am not sure about ccp. I havent used that in compose, maybe @therajanmaurya would have an idea about some other library